v1.0.0

donavanbecker opened the pull request:
## [Version 1.0.0](https://github.com/OpenWonderLabs/homebridge-switchbot/releases/tag/v1.0.0) (2020-11-13)

### Changes

- Offical release of homebridge-Switchbot, which combines both BLE and OpenAPI into 1 plugin.
- Adds Light Sensors to Curtains
    - with iOS 15.1 you can set automations on light sensors.
- Adds Motion Sensor to Contact Sensors
- Adds Support Color Bulbs

donavanbecker committed:
getting ready for color bulbs

donavanbecker committed:
Update plugs.ts

donavanbecker committed:
fix plugin debug logs

yjeanrenaud committed:
added brightness-meter, which returns "dim" or "bright" as a separate light bulb accessory.

beta. added brightness-meter, which returns "dim" or "bright" as a separate light bulb accessory.

Co-authored-by: Donavan Becker <donavan.becker@icloud.com>

donavanbecker committed:
add lightsensor

donavanbecker committed:
Update curtains.ts

donavanbecker committed:
Update contact.ts

donavanbecker committed:
Change from @abandonware/noble to @homebridge/noble

Change from @abandonware/noble to @homebridge/noble

donavanbecker committed:
Update issue-validate.yml

donavanbecker committed:
dependency updates

donavanbecker committed:
Update dependabot.yml

donavanbecker committed:
update workflow, templates, dependants

donavanbecker committed:
Create codeql-analysis.yml

donavanbecker committed:
more updates and potential fix for motion sensor on openapi

donavanbecker committed:
log fixes

donavanbecker committed:
Update dependencies

donavanbecker committed:
fix contact sensors

donavanbecker committed:
update logs

donavanbecker committed:
Update contact.ts

donavanbecker committed:
refractoring

donavanbecker committed:
try latest node-switchbot beta

donavanbecker committed:
more corrections for contact and motion for BLE support

donavanbecker committed:
more changes

donavanbecker committed:
potential fix for #43

donavanbecker committed:
remove issue validation

donavanbecker committed:
removing deviceName from uuid

donavanbecker committed:
Update platform.ts

donavanbecker committed:
Update platform.ts

donavanbecker committed:
more fixes for motion and contact sensors for BLE

donavanbecker committed:
update dependencies

donavanbecker committed:
update dependencies

donavanbecker committed:
Update config.schema.json

donavanbecker committed:
fixes

donavanbecker committed:
fixes

donavanbecker committed:
add UUID to debug log of uuid Creation

donavanbecker committed:
add logging for updateCharacteristic

donavanbecker committed:
Update contact.ts

donavanbecker committed:
fix renames

donavanbecker committed:
is this going to fix contact sensors?

donavanbecker committed:
add displayName to logs and potential fix for airconditioners

donavanbecker committed:
fix naming of devices

donavanbecker committed:
Update tvs.ts

donavanbecker committed:
more fixes

donavanbecker committed:
lower refreshRate requirements

bastiengrignon commented on config.schema.json:
```diff
@@ -47,9 +47,24 @@
-            "description": "Turn plugin's debug mode on."
+            "title": "Logging Setting",
+            "type": "string",
+            "description": "Enble 'Device' or 'Debug' logging.",
```
You're missing an 'a' here 👀 
```suggestion
            "description": "Enable 'Device' or 'Debug' logging.",
```

bastiengrignon commented on src/devices/bulb.ts:
```diff
@@ -42,110 +87,158 @@ export class Bulb {
+
 
-  private statusCode(push: AxiosResponse<any>) {
+  private statusCode(push: { data: { statusCode: any; }; }) {
```
Could be `number` type instead of `any`
```suggestion
  private statusCode(push: { data: { statusCode: number; }; }) {
```

donavanbecker committed:
Update config.schema.json

Co-authored-by: Bastien Grignon <41306995+bastiengrignon@users.noreply.github.com>

donavanbecker committed:
Alpha 2 Beta (#70)

* publish

* getting closer

* should be ready to push to beta branch.

* config to beta

* alpha publish

* fix publish

* almost ready to push to beta branch

* update node-switchbot

* ble in all

* Update package.json

* Update package.json

donavanbecker committed:
remove alpha extras

donavanbecker committed:
fix

#43

donavanbecker committed:
disable tv power command

#11

donavanbecker committed:
add ir config

donavanbecker committed:
Update config.schema.json

donavanbecker committed:
v1.0.0

## [Version 1.0.0](https://github.com/OpenWonderLabs/homebridge-switchbot/releases/tag/v1.0.0) (2020-11-13)

### Changes

- Offical release of homebridge-Switchbot, which combines both BLE and OpenAPI into 1 plugin.
- Adds Light Sensors to Curtains
    - with iOS 15.1 you can set automations on light sensors.
- Adds Motion Sensor to Contact Sensors
- Adds Support Color Bulbs

donavanbecker committed:
v1.0.0

## [Version 1.0.0](https://github.com/OpenWonderLabs/homebridge-switchbot/releases/tag/v1.0.0) (2020-11-13)

### Changes

- Offical release of homebridge-Switchbot, which combines both BLE and OpenAPI into 1 plugin.
- Adds Light Sensors to Curtains
    - with iOS 15.1 you can set automations on light sensors.
- Adds Motion Sensor to Contact Sensors
- Adds Support Color Bulbs
